pkgs/development/python-modules/pillow-jpls/default.nix

```diff
@@ -0,0 +1,84 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, fetchPypi
+, fetchpatch
+, pythonOlder
+, pytestCheckHook
+, cmake
+, ninja
+, scikit-build-core
+, charls
+, eigen
+, fmt
+, numpy
+, pillow
+, pybind11
+, setuptools
+, pathspec
+, pyproject-metadata
+, setuptools_scm
+}:
+
+buildPythonPackage rec {
+  pname = "pillow-jpls";
+  version = "unstable-2023-12-30";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  rev = "c4c0155e47b17d5a2ab09d81256041eac8fd35b9";
+  src = fetchFromGitHub {
+    owner = "planetmarshall";
+    repo = "pillow-jpls";
+    inherit rev;
+    hash = "sha256-+cLd04t+gCAy0gxXi5lex368Y6/Vu/fLsu4E8zcG654=";
+  };
+
+  postPatch = ''
+    substituteInPlace pyproject.toml --replace '"conan~=2.0.16",' ""
```

Comment:
Could you use `pythonRelaxDeps` or `pythonRemoveDeps` for this (see https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonrelaxdepshook-using-pythonrelaxdepshook)?

Reply:
Unfortunately pythonRemoveDeps doesn't appear to work (this has largely been my experience with pyproject-based packages, but I haven't really investigated why).

Reply:
Yes, I also noticed the `pythonRelaxDepsHook` does not always work. Just thinking it should be applied where already possible.